tests/foreman/cli/test_remoteexecution.py

```diff
@@ -643,7 +645,9 @@ def test_positive_run_concurrent_jobs(self, fixture_vmsetup, module_org):
         ids=['rhel7', 'rhel7_fips'],
         indirect=True,
     )
-    @pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url')
+    @pytest.mark.skipif(
+        (not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url'
```

Comment:
Optional:  Here `repos_hosting_url` can be uppercase to keep consistency **or** everywhere it should be updated to lowercase.

Reply:
Definitely lowercase, though I'm not too concerned about enforcing that within this PR.

Reply:
yeah i'm not interested in enforcing case with this pr